tests/test_s3_specific.py

```diff
@@ -39,7 +43,7 @@ def test_transfer_config(s3_rig, tmp_path):
         assert client.s3.download_config == transfer_config
 
     # upload
-    p2 = s3_rig.create_cloud_path("dir_0/file0_0_uploaded.txt")
+    p2 = s3_rig.create_cloud_path(f"{s3_rig.test_dir}/dir_0/file0_0_uploaded.txt")
```

Comment:
Same as previous comment that `s3_rig.create_cloud_path` should already handle `test_dir`.